lib/reltool/test/reltool_server_SUITE.erl

```diff
@@ -2545,11 +2546,22 @@ undefined_regexp(_Config) ->
                                                 {add, ["^priv"]}}]}]}])),
     ok.
 
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%% Checks that reltool_utils can correctly read Windows ERL_LIBS
+
+windows_erl_libs(_Config) ->
+    WinErlLibs =
+        "C:\Program Files\Erlang Libs;C:\Program Files\More Erlang Libs",
```

Comment:
This isn't essential for the test case, but I think there should really be double backslashes here.

Reply:
Of course! Edited and committed it using online editor. Thanks for the feedback!